src/Blef.GameLogic/PokerHands/Flush.cs

```diff
@@ -14,6 +14,11 @@ public override bool IsOnTable(Table table)
             return table.HasSuit(suit);
         }
 
-        protected override int Value => 1000000 * GetSuitValue(suit);
+        protected override int GenericPokerHand => 8;
+
+        protected override int CompareWithinSameGenericPokerHand(PokerHand otherPokerHand)
+        {
+            return 0;
```

Comment:
We wszystkich ukladach kolor nie ma znaczenia podczas porownywania tego samego ukladu, ALE czy nie ma wyjatku od tej reguly gdy ukladem jest kolor? Czy w tym wypadku nie jest tak ze porownujesz ktory kolor powinien byc mocniejszy? P.S. Komentarz ten jest abstrahując od zasadnosci calego PR.